model-optimizer/mo/utils/ir_reader/layer_to_class.py

```diff
@@ -195,6 +195,13 @@ def groupconv_to_conv(op: Node):
             'Weight shape and calculated shape mismatch in GroupConv node {}.'.format(op.name)
     # we need to set this attrs for correct shape infer as convolution
     op['group'] = group
+    # The only way GroupConvolution with 'group' = 1 appears in IR is by converting from TF DepthwiseConv2dNative.
+    # In this case we need to specify 'op' parameter for the
+    # extensions.back.ConvolutionNormalizer.ConvolutionWithGroupsResolver to work properly.
+    # Otherwise  there will be 'Convolution' instead 'GroupConvolution' in restored IR, since 'GroupConvolution' is
+    # extended as node with 'type' = 'Convolution' by IR reader
+    if group == 1:
+        op['op'] = 'DepthwiseConv2dNative'
```

Comment:
It seems like a workaround. I think it is better to change this op value to `GroupConvolution` in the final step of convolution normalizer transformation.

Reply:
`op` value are not saved in IR, so IR Reader does not know this value, if we do not specify it explicitly